std/hash/poseidon/poseidon.go

```diff
@@ -77,7 +76,7 @@ func permutation(api frontend.API, state []frontend.Variable) []frontend.Variabl
 func Poseidon(api frontend.API, input ...frontend.Variable) frontend.Variable {
 	inputLength := len(input)
 	// No support for hashing inputs of length less than 2
```

Comment:
This comment should be revised

Reply:
@HSG88 There is a failed test case "TestPoseidon256".

Reply:
I resolved the above comments in the new commit